src/tests/create-user-test.ts

```diff
@@ -66,5 +66,155 @@ describe('Creating user test', () => {
             await prisma.user.deleteMany();
         })
     })
+
+    it('Trying to create an invalid password, should return error', async () => {
```

Comment:
```suggestion
    it('should return error when trying to create a user with an invalid password', async () => {
```

Acho que assim fica mais no padrão que escrevemos os enunciados. Vale pros outros testes tbm.